sdk/portalservices/Azure.ResourceManager.PortalServicesCopilot/samples/Generated/Samples/Sample_CopilotSettingsResource.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Identity;
+using Azure.ResourceManager.PortalServicesCopilot.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.PortalServicesCopilot.Samples
+{
+    public partial class Sample_CopilotSettingsResource
+    {
+        [Test]
+        [Ignore("Only validating compilation of examples")]
+        public async Task Get_GetCopilotSettings()
+        {
+            // Generated from example definition: 2024-04-01-preview/CopilotSettings_Get.json
+            // this example is just showing the usage of "CopilotSettingsResource_Get" operation, for the dependent resources, they will have to be created separately.
+
+            // get your azure access token, for more details of how Azure SDK get your access token, please refer to https://learn.microsoft.com/en-us/dotnet/azure/sdk/authentication?tabs=command-line
+            TokenCredential cred = new DefaultAzureCredential();
+            // authenticate your client
+            ArmClient client = new ArmClient(cred);
+
+            // this example assumes you already have this CopilotSettingsResource created on azure
+            // for more information of creating CopilotSettingsResource, please refer to the document of CopilotSettingsResource
+            ResourceIdentifier copilotSettingsResourceId = CopilotSettingsResource.CreateResourceIdentifier();
```

Comment:
We have renamed `CopilotSettingsResource` to `PortalServicesCopilotSettingsResource` in generated code.
@ArcturusZhang Any idea why the generated sample did not get renamed accordingly?

Reply:
need to manually remove the legacy sample, regen did not clear the samples